voxel51/users/api.py

```diff
@@ -730,6 +730,19 @@ def delete_job(self, job_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
+    def kill_job(self, job_id):
+        '''Force kills the job with the given ID, which must not have been started.
```

Comment:
`Force kills the job with the given ID. Only available for jobs in the queued or scheduled states.`